README.md

```diff
@@ -484,7 +484,8 @@ oneview_enclosure_group 'EnclosureGroup_1' do
   client <my_client>
   data <resource_data>
   logical_interconnect_groups ['LIG_name1', { name: 'LIG_name2', enclosureIndex: 1 }]
-  action [:create, :create_if_missing, :delete]
+  script # String - The script to be executed
```

Comment:
You just pass the script property alone?

Reply:
@tmiotto Yeah! It's just a string.

Reply:
Yeah, it is a String... Where is the String?

Reply:
@tmiotto This way, is it better?